notes/handbook.sop.weekly-planning.md

```diff
@@ -20,6 +20,17 @@ This describes Dendron's weekly planning process. Think of it as sprint retrospe
   - create `user.{username}.weekly.{date}.plan`
     - this will create a note with the [[Weekly Planning User Checklist|dendron://dendron.handbook/templates.weekly-planning.user]] ^QVIJ3imkWAEo
     - fill out the checklist prior to the meeting
+#### For Weekly Goals
+1. Create a task note for each weekly goal ^ln5zq9jphkjb
+        Note: Even if there is an existing task note, create a new one specific to the weekly goal relating to the task
+1. Every task note associated with a goal should be tagged with #kind.goal
+1. Add a due date for sometime before the next weekly planning (Tuesday of next week)
+1. Ensure the PR link is updated in the fm (where applicable)
```

Comment:
if we're planning on tracking via airtable, then let's add a step here about exporting the task to airtable

Reply:
agreed. updated!